mypy/messages.py

```diff
@@ -991,6 +992,171 @@ def untyped_decorated_function(self, typ: Type, context: Context) -> None:
+                self.note(self.pretty_callable(item), context, offset=2 * OFFSET)
+            if len(tp.items()) > MAX_ITEMS:
+                self.note('<{} more overload(s) not shown>'.format(len(exp.items()) - MAX_ITEMS),
+                          context, offset=2 * OFFSET)
+
+        if isinstance(subtype, TupleType):
+            if not isinstance(subtype.fallback, Instance):
+                return
+            subtype = subtype.fallback
+
+        # Report missing members
+        missing = get_missing_members(subtype, supertype)
+        if (missing and len(missing) < len(supertype.type.protocol_members) and
+                len(missing) <= MAX_ITEMS):
+            self.note("'{}' is missing following '{}' protocol member{}:"
+                      .format(subtype.type.name(), supertype.type.name(),
+                              's' if len(missing) > 1 else ''),
+                      context)
+            self.note(', '.join(missing), context, offset=OFFSET)
+
+        # Report member type conflicts
+        conflict_types = get_conflict_types(subtype, supertype)
+        if conflict_types and (not is_subtype(subtype, erase_type(supertype)) or
+                               not subtype.type.defn.type_vars or
+                               not supertype.type.defn.type_vars):
+            self.note('Following member(s) of {} have '
+                      'conflicts:'.format(self.format(subtype)), context)
+            for name, got, exp in conflict_types[:MAX_ITEMS]:
+                if (not isinstance(exp, (CallableType, Overloaded)) or
+                        not isinstance(exp, (CallableType, Overloaded))):
+                    self.note('{}: expected {}, got {}'.format(name,
+                                                               *self.format_distinctly(exp, got)),
+                              context, offset=OFFSET)
+                else:
+                    self.note('Expected:', context, offset=OFFSET)
+                    if isinstance(exp, CallableType):
+                        self.note(self.pretty_callable(exp), context, offset=2 * OFFSET)
+                    else:
+                        assert isinstance(exp, Overloaded)
+                        pretty_overload(exp)
+                    self.note('Got:', context, offset=OFFSET)
+                    if isinstance(got, CallableType):
+                        self.note(self.pretty_callable(got), context, offset=2 * OFFSET)
+                    else:
+                        assert isinstance(got, Overloaded)
+                        pretty_overload(got)
+            if len(conflict_types) > MAX_ITEMS:
+                self.note('<{} more conflict(s) not shown>'
+                          .format(len(conflict_types) - MAX_ITEMS),
+                          context, offset=OFFSET)
+
+        # Report flag conflicts (i.e. settable vs read-only etc.)
+        for name, subflags, superflags in get_all_flags(subtype, supertype):
```

Comment:
Maybe also limit flag conflicts to only two and crop other like above (having more than two is highly unlikely however).